packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.component.tsx

```diff
@@ -144,157 +131,108 @@ const ImmunizationsForm: React.FC<ImmunizationsFormProps> = ({
-              <div className={styles.immunizationsInputContainer}>
-                <label htmlFor="manufacturer">{t('manufacturer', 'Manufacturer')}</label>
-                <div className="omrs-input-group">
-                  <input
-                    className="omrs-input-outlined"
-                    type="text"
-                    id="manufacturer"
-                    style={{ height: '2.75rem' }}
-                    defaultValue={formState.manufacturer}
-                    onChange={(evt) => updateSingle('manufacturer', evt.target.value)}
-                  />
-                </div>
-              </div>
-            </div>
+      <Form onSubmit={handleFormSubmit} data-testid="immunization-form">
+        {hasSequences(formState.sequences) && (
+          <div className={styles.immunizationSequenceSelect}>
+            <Select
+              id="sequence"
+              name="sequence"
+              value={formState.currentDose.sequenceNumber}
+              onChange={onDoseSelect}
+              className="immunizationSequenceSelect"
+              required
+              labelText={t('sequence', 'Sequence')}>
+              <SelectItem text={t('pleaseSelect', 'Please select')} value="DEFAULT">
+                {t('pleaseSelect', 'Please select')}
+              </SelectItem>
+              {formState.sequences.map((s) => {
+                return (
+                  <SelectItem key={s.sequenceNumber} text={s.sequenceLabel} value={s.sequenceNumber}>
+                    {t(s.sequenceLabel, s.sequenceLabel)}
+                  </SelectItem>
+                );
+              })}
+            </Select>
           </div>
-        </SummaryCard>
-        <div
-          className={
-            enableCreateButtons || enableEditButtons
-              ? `${styles.buttonStyles} ${styles.buttonStylesBorder}`
-              : styles.buttonStyles
-          }>
-          <button
-            type="button"
-            className="omrs-btn omrs-outlined-neutral omrs-rounded"
-            style={{ width: '50%' }}
-            onClick={closeForm}>
-            {t('cancel', 'Cancel')}
-          </button>
-          <button
-            type="submit"
-            style={{ width: '50%' }}
+        )}
+        <div className={styles.immunizationSequenceSelect}>
+          <DatePicker
+            id="vaccinationDate"
+            className="vaccinationDate"
+            maxDate={new Date().toISOString()}
+            dateFormat="d/m/Y"
+            datePickerType="single"
+            value={formState.vaccinationDate}
+            onChange={([date]) => updateSingle('vaccinationDate', date)}>
+            <DatePickerInput
+              id="date-picker-calendar-id"
+              placeholder="dd/mm/yyyy"
+              labelText={t('vaccinationDate', 'Vaccination Date')}
+              type="text"
+            />
+          </DatePicker>
+        </div>
+        <div className={styles.immunizationSequenceSelect}>
+          <DatePicker
```

Comment:
I believe the expiration date should be some date in the future

`minDate={new Date().toISOString()}`